homeassistant/components/blink/services.py

```diff
@@ -3,171 +3,85 @@
 
 import voluptuous as vol
 
-from homeassistant.config_entries import ConfigEntry, ConfigEntryState
-from homeassistant.const import (
-    ATTR_DEVICE_ID,
-    CONF_FILE_PATH,
-    CONF_FILENAME,
-    CONF_NAME,
-    CONF_PIN,
-)
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.const import CONF_PIN
 from homeassistant.core import HomeAssistant, ServiceCall
 from homeassistant.exceptions import HomeAssistantError, ServiceValidationError
-import homeassistant.helpers.config_validation as cv
-import homeassistant.helpers.device_registry as dr
+from homeassistant.helpers import config_validation as cv, issue_registry as ir
 
-from .const import (
-    DOMAIN,
-    SERVICE_REFRESH,
-    SERVICE_SAVE_RECENT_CLIPS,
-    SERVICE_SAVE_VIDEO,
-    SERVICE_SEND_PIN,
-)
-from .coordinator import BlinkUpdateCoordinator
+from .const import ATTR_CONFIG_ENTRY_ID, DOMAIN, SERVICE_REFRESH, SERVICE_SEND_PIN
 
 SERVICE_UPDATE_SCHEMA = vol.Schema(
     {
-        vol.Required(ATTR_DEVICE_ID): vol.All(cv.ensure_list, [cv.string]),
-    }
-)
-SERVICE_SAVE_VIDEO_SCHEMA = vol.Schema(
-    {
-        vol.Required(ATTR_DEVICE_ID): vol.All(cv.ensure_list, [cv.string]),
-        vol.Required(CONF_NAME): cv.string,
-        vol.Required(CONF_FILENAME): cv.string,
+        vol.Required(ATTR_CONFIG_ENTRY_ID): vol.All(cv.ensure_list, [cv.string]),
```

Comment:
This also needs to be reverted if we're keeping the update service as is.

Reply:
Actually, had to put back a lot more code that was removed.